src/ralph/backends/database/es.py

```diff
@@ -47,7 +47,7 @@ def to_documents(self, stream, get_id):
                 "_source": item,
             }
 
-    def put(self, chunk_size=500):
+    def put(self, chunk_size=500, force=False):
```

Comment:
`silent_errors` or `ignore_errors` should be more relevant IMO. I understand `force` like a `git push -f` and it's not the same meaning for me.

Reply:
When forcing a `git push`, it means push this regardless of potential errors occurring without the `--force` flag. So it's basically the same to me.

Reply:
No with `git push --force` you bypass a security, you force to overwrite a remote ref that is not an ancestor of the local ref. It has nothing to do with ignoring errors.

Reply:
After IRL conversation we decided to introduce a new `ignore-errors` option to the `push` command.